src/main/java/seedu/budgetbuddy/Parser.java

```diff
@@ -865,7 +839,8 @@ public Command parseCommand(ExpenseList expenses, SavingList savings, SplitExpen
         }
         
         if (isSetBudgetCommand(input)) {
-            return handleSetBudgetCommand(expenses, input);
+            CommandCreator commandCreator = new SetBudgetCommandCreator(expenses, input);
+            return commandCreator.createCommand();
         }
 
         if (isListBudgetCommand(input)){
```

Comment:
The ListBudgetCommand does not seem to have been seperated into the relevant CommandCreator classes. Would it be better to do so in order for all of us to globally use createCommand()?